modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm

```diff
@@ -56,17 +56,15 @@
 	ears = /obj/item/radio/headset/headset_medsec
 	uniform = /obj/item/clothing/under/rank/security/peacekeeper/security_medic
 	gloves = /obj/item/clothing/gloves/color/latex/nitrile
-	shoes = /obj/item/clothing/shoes/combat/peacekeeper
+	shoes = /obj/item/clothing/shoes/jackboots
 	glasses = /obj/item/clothing/glasses/hud/health/sunglasses
 	suit = /obj/item/clothing/suit/armor/vest/peacekeeper/security_medic
 	l_hand = /obj/item/storage/firstaid/brute
 	head = /obj/item/clothing/head/beret/sec/peacekeeper/security_medic
 
-	backpack_contents = list(/obj/item/melee/baton/telescopic, /obj/item/storage/box/gunset/security_medic)
```

Comment:
Same with what you did before, give them something to supplement this if you're gonna remove it. The SecMed has authority to act like an officer if need be, they just shouldn't be. Removing any tools for self-defence, or to make arrests is just dumb.

Reply:
I will give them a disabler. They will need to go to the locker for a baton, however.